### PR TITLE
Allocate enums for Qualcomm

### PR DESCRIPTION
**Pull request by MatAshish**

### CODE REVIEW

File: xml/gl.xml [hunk 1]
```diff
@@ -7017,6 +7017,12 @@ typedef unsigned int GLhandleARB;
             <unused start="0x9700" end="0x970F" vendor="Samsung"/>
     </enums>
 
+    <enums namespace="GL" start="0x9710" end="0x971F" vendor="Qualcomm" comment="Reserved for Ashish Mathur">
+			<enum value="0x9710" name="GL_TEXTURE_Y_DEGAMMA_QCOM" group="TexParameterI,GetTexParameter"/>
+			<enum value="0x9711" name="GL_TEXTURE_CBCR_DEGAMMA_QCOM" group="TexParameterI,GetTexParameter"/>
```
Comment (SunSerega) [inline xml/gl.xml:7022]: rogue tabs
```suggestion
            <enum value="0x9710" name="GL_TEXTURE_Y_DEGAMMA_QCOM" group="TexParameterI,GetTexParameter"/>
            <enum value="0x9711" name="GL_TEXTURE_CBCR_DEGAMMA_QCOM" group="TexParameterI,GetTexParameter"/>
```

Comment (MatAshish) [reply xml/gl.xml]: Sorry about that. Thanks for reviewing.

Comment (SunSerega) [reply xml/gl.xml]: I think there are still tabs, but now less of them.

File: xml/gl.xml [hunk 2]
```diff
@@ -7017,6 +7017,12 @@ typedef unsigned int GLhandleARB;
             <unused start="0x9700" end="0x970F" vendor="Samsung"/>
     </enums>
 
+    <enums namespace="GL" start="0x9710" end="0x971F" vendor="Qualcomm" comment="Reserved for Ashish Mathur">
+        <enum value="0x9710" name="GL_TEXTURE_Y_DEGAMMA_QCOM" group="TexParameterI,GetTexParameter"/>
```
Comment (MatAshish) [inline xml/gl.xml:7021]: Replaced tabs with spaces.

**Comment by oddhack** [comment]
LGTM. Is this ready to merge?

**Comment by MatAshish** [comment]
Can this be accepted onto the main branch ?

**Comment by oddhack** [comment]
> Can this be accepted onto the main branch ?

Yes, sorry. We just had a face to face Khronos meeting and not a lot gets done the week after that.

**Comment by SunSerega** [comment]
Everywhere else in XML the `vendor="QCOM"` is used, but here `vendor="Qualcomm"` was added.

I can make a quick PR to make it the same everywhere, but which of the 2 would you prefer to keep, @MatAshish?

**Comment by MatAshish** [comment]
@SunSerega : Thanks for pointing out.  I based vendor as "Qualcomm" on a previous check-in in the same file for GL_FRAMEBUFFER_FETCH_NONCOHERENT_QCOM. Looks like it was a mistake and we can take "QCOM" to be consistent. I will make another pull request with TextureParameterName and QCOM.

**Pull request merged**